src/pages/authentication/building-with-angular.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Building an app with Angular
```

Comment:
I think we need to expose this tutorial more. Right now, it's hard to find - you'd basically need to know you have to click on "auth" and then find it in the list of tutorials. I'd suggest two things:

- Update the auth overview file and link to the tutorials available (the todo and this) using `[@page-reference | inline]`. This would require setting an image in the file meta (uncomment lines 8-11) - we could just use something that already exists.
- Update the build an app page to link to this tutorial (in addition to the links to the todo tutorial)

Reply:
Sounds like you're suggesting some sort of... interconnected web of information 🤔

Reply:
![image](https://user-images.githubusercontent.com/1618764/100995621-52f34180-3558-11eb-850f-381da5b9c32f.png)

Reply:
👍